### PR TITLE
Aet ui style fixes

### PR DESCRIPTION
**Pull request by sabeen**
small improvements on the interface.
Toggleable content for mapping: switching between Mapping rules and the JSON view.

### CODE REVIEW

**Comment by sabeen** [comment]
btw: the way the hint messages work on the entity types and the tests for that seem to be outdated. I think there is no need for a hint like "invalid schema", since we have that error already on the text input side. An invalid schema won't be added to the pipeline. So we only have the case that either an entity type schema has been added or not.

File: aether-ui/aether/ui/assets/apps/components/EntityTypeViewer.jsx [hunk 1]
```diff
@@ -86,7 +86,7 @@ class EntityTypeViewer extends Component {
         <div className='hint'>
           <FormattedMessage
             id='entityTypes.entity.invalid.schema'
-            defaultMessage='Invalid schema'
+            defaultMessage='No Entity Types added to this pipeline yet.'
```
Comment (obdulia-losantos) [inline aether-ui/aether/ui/assets/apps/components/EntityTypeViewer.jsx:89]: This is not a valid error message in this case, it tries to tell the user that nor `name` nor `fields` were included in the entity type schema, but there is an Entity Type.

The error message was not accurate before but if you are changing it... :grimacing:

File: aether-ui/aether/ui/assets/apps/pipeline/sections/Mapping.jsx [hunk 2]
```diff
@@ -29,32 +30,63 @@ class Mapping extends Component {
...
-                <span className='details-title'>
-                  <FormattedMessage
-                    id='mapping.rules.button.ok'
-                    defaultMessage='Apply mapping rules to pipeline'
-                  />
-                </span>
-              </button>
+        <div className='toggleable-content'>
+          <div className='tabs'>
+            <button
```
Comment (obdulia-losantos) [inline aether-ui/aether/ui/assets/apps/pipeline/sections/Mapping.jsx:46]: What happens if someone press twice to the same tab? Is the `toggleMappingView` going to change the current view? Is this the expected behaviour?

File: aether-ui/aether/ui/assets/apps/components/EntityTypeViewer.jsx (hunk 1 above)
Comment (sabeen) [reply aether-ui/aether/ui/assets/apps/components/EntityTypeViewer.jsx:89]: hm… I just wanted it to display this message when there is nothing added yet. I am not sure how to change the whole error behaviour…

File: aether-ui/aether/ui/assets/apps/pipeline/sections/Mapping.jsx (hunk 2 above)
Comment (sabeen) [reply aether-ui/aether/ui/assets/apps/pipeline/sections/Mapping.jsx:46]: 😬

Comment (sabeen) [reply aether-ui/aether/ui/assets/apps/pipeline/sections/Mapping.jsx:46]: uups

File: aether-ui/aether/ui/assets/apps/components/EntityTypeViewer.jsx (hunk 1 above)
Comment (obdulia-losantos) [reply aether-ui/aether/ui/assets/apps/components/EntityTypeViewer.jsx:89]: In that case I can change it later, once you are done with this :wink:

File: aether-ui/aether/ui/assets/apps/pipeline/sections/Mapping.jsx (hunk 2 above)
Comment (obdulia-losantos) [reply aether-ui/aether/ui/assets/apps/pipeline/sections/Mapping.jsx:46]: Gotcha!

File: aether-ui/aether/ui/assets/css/base/_global.scss [hunk 3]
```diff
@@ -171,6 +171,7 @@ textarea:active {
     background: $light-grey;
     border-radius: $border-radius $border-radius 0 0;
     color: $text-color;
+    pointer-events: none;
```
Comment (obdulia-losantos) [inline aether-ui/aether/ui/assets/css/base/_global.scss]: :roll_eyes:

Comment (sabeen) [reply aether-ui/aether/ui/assets/css/base/_global.scss]: 😁

Comment (sabeen) [reply aether-ui/aether/ui/assets/css/base/_global.scss]: too cheesy? It works perfectly…

**Review by obdulia-losantos — APPROVED** [review]
LGTM!

**Comment by obdulia-losantos** [comment]
I think we need to unify the different error messages we can receive/generate from the AVRO parser and our own rules.

**Pull request merged**